File: python3/1541.py
import math

while True:
    entrada = [int(i) for i in input().split()]
    a = entrada[0]
    if(a == 0):
        break
    b = entrada[1]
    c = entrada[2]

    area = a * b
    area_real = math.pow(int(math.sqrt(area)), 2)
    terreno = (area * 100) / c
    saida = int(math.sqrt(terreno))

    print(saida)

File: cpp/beginner/1541.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {

    int a, b, c;
    int area, areaReal, terreno;

    while(cin >> a) {
        if (a == 0) {
            break;
        }
        cin >> b >> c;
        area = a * b;
        areaReal = pow(int(sqrt(area)), 2);
        terreno = (area * 100) / c;
        cout << int(sqrt(terreno))  << endl;
    }

    return 0;
}
